### PR TITLE
Branding Updates

### PR DESCRIPTION
**Pull request by nedo-SAP**

### CODE REVIEW

File: tutorials/cp-cf-security-xsuaa-multi-tenant/cp-cf-security-xsuaa-multi-tenant.md [hunk 1]
```diff
@@ -343,13 +343,13 @@ You'll now see the application with the message `no data` because you have to as
 
 Assign your user the role collection `ProductListViewer` that contains the necessary role to view the products in the product list.
 
-3. Open the SAP Cloud Platform Cockpit.
+3. Open the SAP BTP Cockpit.
```
Comment (MichaelCzcz) [inline tutorials/cp-cf-security-xsuaa-multi-tenant/cp-cf-security-xsuaa-multi-tenant.md:346]: SAP BTP cockpit

File: tutorials/cp-cf-security-xsuaa-create/cp-cf-security-xsuaa-create.md [hunk 2]
```diff
@@ -413,7 +413,7 @@ Your application has two routes that are defined in the `manifest.yml`. The dire
 
 Assign your user the role collection that contains the necessary role to view the products in the product list.
 
-3. Open the SAP Cloud Platform Cockpit.
+3. Open the SAP BTP Cockpit.
```
Comment (MichaelCzcz) [inline tutorials/cp-cf-security-xsuaa-create/cp-cf-security-xsuaa-create.md:416]: SAP BTP cockpit

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]
Please make sure to use the correct spelling of product names.

https://wiki.wdf.sap.corp/wiki/display/NDW/SAP+BTP%3A+Rebranding+Guidelines

File: tutorials/cp-cf-security-xsuaa-multi-tenant/cp-cf-security-xsuaa-multi-tenant.md [hunk 3]
```diff
@@ -320,7 +320,7 @@ cf map-route approuter cfapps.eu10.hana.ondemand.com --hostname consumer-tenant-
 [ACCORDION-BEGIN [Step 8: ](Access the application with the consumer subaccount)]
 To access the application you need to subscribe to it. Follow these steps to subscribe to the SaaS application with the consumer subaccount and call the application URL.
 
-1. Open the [SAP Cloud Platform trial](https://account.hanatrial.ondemand.com/cockpit/#/home/trialhome).
+1. Open the [SAP BTP trial](https://account.hanatrial.ondemand.com/cockpit/#/home/trialhome).
```
[the remark this answers is not in the document]
Comment (nedo-SAP) [reply tutorials/cp-cf-security-xsuaa-multi-tenant/cp-cf-security-xsuaa-multi-tenant.md]: fixed

File: tutorials/cp-cf-security-xsuaa-multi-tenant/cp-cf-security-xsuaa-multi-tenant.md (hunk 1 above)
Comment (nedo-SAP) [reply tutorials/cp-cf-security-xsuaa-multi-tenant/cp-cf-security-xsuaa-multi-tenant.md]: fixed

File: tutorials/cp-cf-security-xsuaa-create/cp-cf-security-xsuaa-create.md (hunk 2 above)
Comment (nedo-SAP) [reply tutorials/cp-cf-security-xsuaa-create/cp-cf-security-xsuaa-create.md]: fixed

**Pull request merged**